client/src/components/AboutPage.js

```diff
@@ -1,11 +1,186 @@
-import React from "react";
-import Page from "./Page";
-import { Container, Row, Col } from "reactstrap";
-import { img } from "./Speaker2.css";
-import hoop from "../images/SVG/hoop.svg";
-import x from "../images/SVG/x.svg";
-import { image } from "../images";
-import ReadMore from './ReadMore'
+import React from 'react';
+import styled from 'styled-components';
+
+import Page from './Page';
+import ReadMore from './ReadMore';
+
+import x from '../images/SVG/x.svg';
+
+import mediaQueryMin from '../styles/MediaQueriesMixin';
```

Comment:
naming; I know its everywhere now :) I just want to say that the Min part is an implementation detail, the name IMO should be `breakpoint`